core/thread/CMakeLists.txt

```diff
@@ -63,9 +63,11 @@ ROOT_STANDARD_LIBRARY_PACKAGE(Thread
     src/TThreadImp.cxx
   STAGE1
   DICTIONARY_OPTIONS
-    -writeEmptyRootPCM 
+    -writeEmptyRootPCM
   DEPENDENCIES
     Core
+  BUILTINS
```

Comment:
Is that dependency conditional on IMT being ON?  (i.e. can you still build ROOT without TBB)

Reply:
Yes, it should be possible to build `Thread` library without `TBB`.
`BUILTINS` argument checked in `ROOT_LINKER_LIBRARY` macro and only if specified target exists add it to dependency for build library. See: https://github.com/root-project/root/blob/master/cmake/modules/RootMacros.cmake#L952-L958